packages/modeling/src/operations/transforms/align.js

```diff
@@ -0,0 +1,87 @@
+const flatten = require('../../utils/flatten')
+const { measureAggregateBoundingBox } = require('../../measurements')
+const { translate } = require('./translate')
+
+const validateOptions = (options) => {
+  const defaults = {
+    modes: ['center', 'center', 'lower'],
+    alignTo: [0, 0, 0],
+    grouped: false
+  }
+  options = Object.assign({}, defaults, options)
+
+  if (options.modes.length !== 3) throw new Error('align(): modes must be an array of length 3')
+  if (options.modes.filter(mode => ['center', 'upper', 'lower', 'none'].includes(mode)).length !== 3) throw new Error('align(): all modes must be one of "center", "upper" or "lower"')
```

Comment:
I don't mind 'upper' and 'lower' however we should be consistent with our bounding box definition. It think this will make these options more understandable.

https://en.wikipedia.org/wiki/Minimum_bounding_box

Reply:
Not quite sure what changes this implies.  What are you thinking?

Reply:
Not sure.

We could change measureBoundingBox to refer ‘lower’ and ‘upper’.

Or

We could change align to refer ‘min’ and ‘max’

But there may be another term as well.

Reply:
let's go min,max,center